util/po2php.php

```diff
@@ -92,8 +92,10 @@ function escape_s($match){
 				.preg_replace_callback($escape_s_exp,'escape_s',$match[2]) .",\n";
 		}
 
-		if (substr($l,0,6)=="msgid_") { $ink = False; $out .= '$a->strings["'.$k.'"] = '; };
-
+		if (substr($l, 0, 6) == "msgid_") {
+			$ink = false;
+			$out .= '$a->strings["' . $k . '"] = ';
+		}
 
 		if ($ink) {
 			$k .= trim($l,"\"\r\n");
```

Comment:
Found only 4 occurrences, including this one. I fixed it here only and marked in todos.txt file.